src/DeviceFirmata.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+  DeviceFirmata.cpp - Firmata library
+*/
+
+#include "DeviceFirmata.h"
+#include "utility/Boards.h"
+#include <Base64.h>
```

Comment:
What is the advantage of using Base64 encoding rather than the 7-bit encoding with the existing [Encoder7Bit class](https://github.com/firmata/ConfigurableFirmata/blob/master/src/Encoder7Bit.cpp)?